dlc-messages/Cargo.toml

```diff
@@ -12,8 +12,8 @@ use-serde = ["serde", "secp256k1-zkp/use-serde"]
 
 [dependencies]
 bitcoin = {version = "0.29.2"}
-dlc = {version = "0.4.0", path = "../dlc"}
-lightning = {version = "0.0.116"}
+dlc = {version = "0.4.0", path = "../dlc", features = ["serde"] }
```

Comment:
I think `serde` is supposed to be feature gated (not 100% sure it's correctly setup but at least that was the goal)

Reply:
You are right. We want to only enable `serde` for the `dlc` crate if the `serde` feature flag is passed. I'll fix that.